.travis.yml

```diff
@@ -25,45 +25,50 @@ env:
+  # APP_CREDENTIALS_STAGING_PWD
+  - secure: LMuxrulQKDgLkUa26eZ8oP8nNxEsWUFUdmbqCxpkAmgfhQknxGMsk58ecDKlCNu7bWnAZZl0ZjWVP/VyVA7kpMNEEMFo/kOScOVsOL5mydqXmOYZa//hpiqZj/ip7JUDCQvreOPHnwBh9Be7f+BOeVdw5Go3kmKUWTrgS/5FpYuVAzbw9koMPC2RFEZTtqIFYwnnOkq/FwvpYDzPx9JlkKFkowg2P6XYPI4VCLhlQu8A3x9pM8ZWuxIkmojzdJOtxhKwtVQYQCwmD5YxXHhmOdP0FkTZoPs1kEPn1CC67uHObQaAPoW4FXO+Leb20ctc7TZZxOSMvU678PC/8uveg+vgh6KixzRIB4wT906etEj0Y6f1nE9QojUVpbWCQY1iOx3OnuseGo0amAR+4Cqe+NwAMNPIKR9HcDYodMDh3TcX/AlNT/f2FOzkI+DP6L2eTBuOvg/g88THrYj26WIk8r177Sav1xznCLxK8dUxZJqDHhhtGLZ8nQA3p2VpFewmNzj6OZZ8T60z0UOuXtXCSferZIBcQuVURLklfQCRmz6YG/c4BLkP8Qnw9PpcFera7UpJr1j+vRYq9m/srmlPjxNZBSymdIM//eXZPedmElxzMrNfCysYI09nbMP5YOegd4VmE4DqSIGdVYWoLAFcAEF6Fw5besWxGJneA3O+63s=
+  # APP_CREDENTIALS_PROD_PWD
+  - secure: DFlr4hQAz/btOkHAW+04VSfLIPPxrw6VVsu0OcFc+AhexkeUZEUg72/duUxGgr/b7Nd66tbJWYpPL/LChmsd9xTE2D2H3nVrLYLBy5QFuTIy63aGdELgJujxiSNzSEHjy7yFqSmutCw7SRV/7IW2UmOjc3Be8tBbpcOaUAHOseH2fMRlEmHSH3YRfV6L53IvP/ApHl7ZnWTa6JM7u8IXEpKrJrvWwq7duST5+HY7s+dZKZ6UIEL59z8T5mcEf9fXexjwghIZVimCxRNG5aGIXGi6yifjF7H2KTy3vkhFqthZZ74oseDY4A3p+XL9uW/SY8sE4wupNvnaHwpKTIi9luRa/LrVMZN6jTfQ85HlfkVXHT/AxC0AWQGF+vqo4uGVCs+8PTm5t+xbsqKvJjAbyfVyQqkaqSTTwd4Sbj85cnE09LnEm1tIk78/T8EliHckkkaJKbVyH0gghK5fOw3AdHJ5rkOJ5W+NM1UAexHjF0gcOsR4fWL49zy2UGIirNyQV/t8e/IZS/n9eednNuE4TrxU3Dk4OsXHs4RfSHqXFfmvF4Pi35W8XCMPgCurMOMH7uORlZH9iqIhEtGW4DYhnlePPq/asCBOiiEQkqD6rQC1saY28uTGdRxrIjNUKsvPPqdwvJSoEugnF7v9ikdxIhTsep1yRKKlEd2RT4gMiIM=
 before_script:
 - psql -c 'create database travis_ci_test;' -U postgres
 script:
 - "./travis-build.sh --quiet verify"
 deploy:
 - provider: script
-  script: git fetch --unshallow && git push --force https://govuk-dfe-bat-deployment-user:$AZURE_WA_PASSWORD@bat-dev-manage-courses-api-app.scm.azurewebsites.net:443/bat-dev-manage-courses-api-app.git
-    HEAD:refs/heads/master
+  script: bash deploy.sh dev
   on: master
 - provider: script
-  script: git fetch --unshallow && git push --force https://govuk-dfe-bat-deployment-user:$AZURE_WA_PASSWORD@bat-staging-manage-courses-api-app.scm.azurewebsites.net:443/bat-staging-manage-courses-api-app.git
-    HEAD:refs/heads/master
+  script: bash deploy.sh staging
   on: staging
 - provider: script
-  script: git fetch --unshallow && git push --force https://govuk-dfe-bat-deployment-user:$AZURE_WA_PASSWORD@bat-prod-manage-courses-api-app.scm.azurewebsites.net:443/bat-prod-manage-courses-api-app.git
-    HEAD:refs/heads/master
+  script: bash deploy.sh prod
   on: production
 - provider: script
-  script: "./travis-deploy-sc-exporter.sh govuk-dfe-bat-deployment-user:$AZURE_WA_PASSWORD
+  script: "./travis-deploy-sc-exporter.sh $bat-dev-manage-courses-api-app:$APP_CREDENTIALS_DEV_PWD
```

Comment:
`$bat` needs escaping here -> `\$bat`

Reply:
Here and all the instances of `$bat` below.

Reply:
Yes, it did need it